docroot/modules/humsci/hs_migrate/hs_migrate.module

```diff
@@ -129,3 +129,71 @@ function _hs_migrate_media_imported($media_id) {
 
   return FALSE;
 }
+
+/**
+ * Deletes importer configuration and drops migration tables.
+ *
+ * This function takes either a single string or an array of importer names as
+ * strings, removes their configuration, migration tables, and logs the results.
+ *
+ * @param string|string[] $input
+ *   A single string or an array of importer names to process.
+ *
+ * @throws \InvalidArgumentException
+ *   When the input is neither a string nor an array of strings.
+ */
+function _hs_migrate_remove_importers(string|array $input): void {
```

Comment:
Because this is a helper function specifically for database updates it should be moved from here to the `.install` file. Sorry did not notice this the first time I approved it.

Reply:
@joegl we'll have to move this back since Seth has departed. Let me know if you'd like me to update the PR.

Reply:
I think I got it but will let you know. Thanks

Reply:
Moved